task_3.py

```diff
@@ -0,0 +1,22 @@
+"""
+Задайте список размерностью N. Каждый элемент списка вычисляется выражением (1 + 1 / n) ** n,
+где n – позиция (не индекс) элемента в списке, причем 1 < n < N.
+Выведите сумму элементов списка. Ответ округлите до сотых.
+
+Ввод: значение типа <int>
+Вывод: значение типа <float>
+
+Пример:
+1
+2.0
+
+2
+4.25
+
+3
+6.62
+"""
+
+n = int(input())
+lst = [round((1+1/i)**i, 3) for i in range(1, n+1)]
```

Comment:
Не забывайте про PEP8.
```suggestion
lst = [round((1 + 1 / i) ** i, 3) for i in range(1, n + 1)]
```